{"name": "test_answer", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 5\n +  where 4 = inc(3)", "trace": "def test_answer():\n>       assert inc(3) == 5\nE       assert 4 == 5\nE        +  where 4 = inc(3)\n\njzwpytest\\test_pytest.py:13: AssertionError"}, "attachments": [{"name": "stdout", "source": "404badbc-8ede-4ed4-a2b6-99ee9f863414-attachment.txt", "type": "text/plain"}], "start": 1589008883123, "stop": 1589008883124, "uuid": "b90dc0aa-591c-4a7f-98d6-db79774f886c", "historyId": "1475c044458f7d5c83d33aa54e5fc40a", "testCaseId": "8b82c10f1fa0be2614a753e7b79aa85b", "fullName": "jzwpytest.test_pytest#test_answer", "labels": [{"name": "parentSuite", "value": "jzwpytest"}, {"name": "suite", "value": "test_pytest"}, {"name": "host", "value": "jiangzhw0100"}, {"name": "thread", "value": "26168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "jzwpytest.test_pytest"}]}